### DISCUSSION TITLE
編集履歴から過去のリビジョンを復元する機能がほしい

### DISCUSSION THREAD

**Discussion by munierujp** [increments/qiita-discussions#347] [要望・アイデア・不具合]
「記事を更新したものの、やっぱり前のほうがよかった」といった場合に、過去のリビジョンをボタン一発で復元できると便利だと思います。

**Comment by tanaka-asami415** [comment]
@munierujp 
ご意見ありがとうございます！
確かにそのような場面は多そうなので、検討いたします🙇

**Comment by acronhub** [comment]
@munierujp 
大変長らくお待たせいたしました 🙇 
ご所望の機能とは異なるかとは思いますが、代替機能としてご使用いただける機能をリリースいたしました 🚀 

記事の編集履歴からMarkdownの閲覧及びコピーを行う機能でございます。
詳細は [リリースノート](https://qiita.com/release-notes/2025-05-16) をご参照くださいませ。

引き続き過去のリビジョンを簡単に復元できる機能の開発も検討していきます。

**Reply by munierujp** [reply]
ありがとうございます🥰
